src/ir_Funiki.h

```diff
@@ -0,0 +1,204 @@
+// Copyright 2024 QuangThai2297
+
+/// @file
+/// @brief Support for FUNIKI A/C protocols.
+/// @see https://github.com/crankyoldgit/IRremoteESP8266/issues/2112
+
+
```

Comment:
Please add the supported devices info here please. It's required for our automatic documentation.
e.g. Like the following for Kelvinator.
https://github.com/crankyoldgit/IRremoteESP8266/blob/dc0fd31706ff26fbd28b004a971ea4a702fb5e60/src/ir_Kelvinator.h#L5-L19

Reply:
Thank you! Let me check!